scipy/stats/tests/test_kdeoth.py

```diff
@@ -413,6 +413,15 @@ def test_seed_sub(gkde_trail):
         samp2 = gkde_trail.resample(n_sample, seed=rstate2)
         assert_allclose(samp1, samp2, atol=1e-13)
 
+        # check that np.random.Generator can be used (numpy >= 1.17)
```

Comment:
same comment as up in `common_tests.py`, this cannot  fail the test, not clear to me what is the intent on these lines of code.

Reply:
The purpose of the test is to perform a smoke test of whether a `Generator` will work when given to `gkde_trail.resample`. If Generator isn't available (old numpy) then the test is skipped, but if it is available then it should be run. Perhaps a better way is:
```
hasattr(np.random, 'Generator'):
    rng = np.random.default_rng(1234)
    gkde_trail.resample(n_sample, seed=rng)
```

Reply:
yeah see comment above, let's call this a non-blocking comment and hopefully someone finds a way to add a test in future PR.